### PR TITLE
feature/complete-the-event-details-screen-and-showing-all-data

### PR DESCRIPTION
**Pull request by mhlangagc**

### CODE REVIEW

File: Big Neon UI/Cells/Explore/ExploreUpcomingCell.swift [hunk 1]
```diff
@@ -73,9 +42,8 @@ public class UpcomingEventCell: UICollectionViewCell {
         return label
     }()
     
-    private let priceView: EventPriceView = {
+    public let priceView: EventPriceView = {
         let view = EventPriceView()
```
Comment (Krakaw) [inline Big Neon UI/Cells/Explore/ExploreUpcomingCell.swift]: I'ave added `event.priceTag()` which gives the range, the label just needs to expand to accept it.

File: Big Neon Core/Models/Event/Event.swift [hunk 2]
```diff
@@ -24,7 +24,34 @@ public struct Event: Codable {
     public let externalURL: String?
     public let userIsInterested: Bool
     public let localizedTimes: LocalizedTimes
-    
+
+    public func compressImage(url: String, quality: String = "low") -> String {
```
Comment (Krakaw) [inline Big Neon Core/Models/Event/Event.swift]: Maybe this needs to move to a static function / a helper method. But you can call this to get the compressed image urls

Comment (mhlangagc) [reply Big Neon Core/Models/Event/Event.swift]: Awesome. I'll see where to put it. 👍

File: Big Neon UI/Cells/Explore/ExploreUpcomingCell.swift (hunk 1 above)
Comment (mhlangagc) [reply Big Neon UI/Cells/Explore/ExploreUpcomingCell.swift]: Sweet!! I'll implement that. 💯

**Pull request merged**